### DISCUSSION TITLE
Destroyed Buildings

### DISCUSSION THREAD

**Discussion by OverlordZorn** [Vdauphin/HeartsAndMinds#1501] [Question and answer] [answered]
How are destroyed buildings being handled?

Is there some save state for those built into Hearts and Minds?

Im looking for a way to long term destroy some buildings in a town to simulate a desaster but it would be a buzz killing if those buildings are repaired after a save+restart.

I tried to destroy some buildings before with editor modules but it just ended up with the same building repaired on top of the wreck.

**Comment by Vdauphin** [comment, marked as answer]
Hello,

> Is there some save state for those built into Hearts and Minds?

No, the mission just listen to [addMissionEventHandler ["BuildingChanged", btc_rep_fnc_buildingchanged];](https://github.com/Vdauphin/HeartsAndMinds/blob/0042d7ebd9e33ba5be58bb07e44c3e4e749db080/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/eh/server.sqf#L22)
and change reputation accordingly: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/rep/buildingchanged.sqf

> I tried to destroy some buildings before with editor modules but it just ended up with the same building repaired on top of the wreck.

BI issue 🤷: https://feedback.bistudio.com/T133265
